libbuteosyncfw/common/TransportTracker.cpp

```diff
@@ -175,19 +177,17 @@ void TransportTracker::updateState(Sync::ConnectivityType aType,
     }
     if(oldState != aState)
     {
-        if (aType != Sync::CONNECTIVITY_INTERNET) {
+        if (aType != Sync::CONNECTIVITY_INTERNET)
+        {
             emit connectivityStateChanged(aType, aState);
         }
-        else {
```

Comment:
The  "emit networkStateChanged" is called from "onInternetStateChanged" because it has different arguments.